helm/cluster-vsphere/templates/default-helmrepository.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Capabilities.APIVersions.Has "source.toolkit.fluxcd.io/v1beta2/HelmRepository" -}}
```

Comment:
Do we need that? I think it may have a really nasty consequences leading to hours of debugging. If anything I'd add something like `.Values.coreApps.enabled` to be able to disable all of that. But let's have someone who needs this first because that will be basically an untested functionality.

Reply:
ok, let's wait till someone needs it